index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/style.css" />
+    <title>JS Conditional Code</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <header>JS Conditional Code</header>
+      <div class="nav-links">
+        <a href="#Introduction" class="nav-link">Introduction</a>
+        <a href="#Conditional_Code_Format" class="nav-link"
+          >Conditional Code Format</a
+        >
+        <a href="#Conditional_Operators" class="nav-link"
+          >Conditional Operators</a
+        >
+        <a href="#Equals_Signs" class="nav-link">Equals Signs</a>
+        <a href="#Nested_Conditions" class="nav-link">Nested Conditions</a>
+        <a href="#And/Or_Operators" class="nav-link">And/Or Operators</a>
+        <a href="#Arithmetic_Operators" class="nav-link"
+          >Arithmetic Operators</a
+        >
+        <!--
+            <a href="#The_Remainder_Operator" class="nav-link">The Remainder Operator</a>
+            <a href="#Arithmetic_Shorthand" class="nav-link">Arithmetic Shorthand</a>
+            <a href="#Unary_(++)_Operators" class="nav-link">Unary (++) Operators</a>
+            <a href="#Ternary_Operators" class="nav-link">Ternary Operators</a>
+            -->
+      </div>
+    </nav>
+
+    <main id="main-doc">
```

Comment:
Please note that classes are better to work with CSS, ids should be used only where necessary.
Read more about why classes are better than ids [here](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4).